### PR TITLE
Dummy PR so we can review changes since last upstream change

### PR DESCRIPTION
**Pull request by coretl**

### CODE REVIEW

File: src/actions/fetch/copier.test.ts [hunk 1]
```diff
@@ -75,7 +75,6 @@ describe('fetch:copier', () => {
 
   const mockReader: UrlReader = {
     readUrl: jest.fn(),
-    read: jest.fn(),
```
Comment (coretl) [inline src/actions/fetch/copier.test.ts:78]: Why is this deleted?

File: src/actions/fetch/copier.ts [hunk 2]
```diff
@@ -72,6 +71,18 @@ export class CopierRunner {
...
+      templateContentsDir,
+    );
+
+    const copierInfo = {
+      ...copierJson,
+      ...values,
+    };
+
+    await fs.writeJSON(path.join(templateDir, 'copier.json'), copierInfo);
+
```
Comment (coretl) [inline src/actions/fetch/copier.ts]: I don't understand what the `copier.json` is there for. Looks like a leftover from the cookiecutter on that made a json file in the root... Can this be deleted?

File: src/actions/fetch/copier.ts [hunk 3]
```diff
@@ -182,17 +209,6 @@ export function createFetchCopierAction(options: {
...
-        ctx.input.copyWithoutRender &&
-        !Array.isArray(ctx.input.copyWithoutRender)
-      ) {
-        throw new InputError(
-          'Fetch action input copyWithoutRender must be an Array',
-        );
-      }
-      if (ctx.input.extensions && !Array.isArray(ctx.input.extensions)) {
-        throw new InputError('Fetch action input extensions must be an Array');
-      }
```
Comment (coretl) [inline src/actions/fetch/copier.ts:195]: What was this code doing and why can it now be deleted?

File: .github/workflows/ci.yml
Comment (coretl) [inline .github/workflows/ci.yml:1]: Will there be an npm publish action added to this file?

File: src/actions/fetch/copier.test.ts (hunk 1 above)
Comment (M-Chan) [reply src/actions/fetch/copier.test.ts:78]: it's no longer a parameter for UrlReader and isn't used

File: src/actions/fetch/copier.ts (hunk 3 above)
Comment (M-Chan) [reply src/actions/fetch/copier.ts:195]: It was not doing anything. They just copied over _half_ of the code from the cookie-cutter one so this never actually worked. They never copied all the parameters for ctx.input so those parameters never existed hence this code never worked.

File: .github/workflows/ci.yml
Comment (M-Chan) [reply .github/workflows/ci.yml:1]: Never really thought of that. Would be neat to automate it though. If you'd like, I'd look into it?

File: src/actions/fetch/copier.ts (hunk 2 above)
Comment (M-Chan) [reply src/actions/fetch/copier.ts]: I thought this was the way to read the template.yaml (or in our case the catalog-info.yaml) file to see what variables are needed. Not completely sure though. Once I'm back working on the containers part (after this EPICS training week, if that's okay), I'll try deleting this part of the code and test to see if it still works / is needed

File: .github/workflows/ci.yml
Comment (coretl) [reply .github/workflows/ci.yml:1]: Yes please, this page has details:
https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-using-yarn

I'll setup a quick meeting with @gfrn, @PeterC-DLS and @garryod to discuss scoping of the token and where it is stored

Comment (PeterC-DLS) [reply .github/workflows/ci.yml:1]: Note that we publish only on a version tag: https://github.com/DiamondLightSource/davidia/blob/main/.github/workflows/npm-publish.yaml

File: src/actions/fetch/copier.ts (hunk 2 above)
Comment (M-Chan) [reply src/actions/fetch/copier.ts]: okay yeah this is not needed
plugin works as required without this section
deleting this section even allows me to simplify other sections too to suit our usages
